spec/acceptance/lastEvaluatedKeyWithFilterAndPagingTest.spec.ts

```diff
@@ -174,6 +179,7 @@ describe('lastEvaluatedKeyWithFilterAndPagingTest', function ()
         assert.deepEqual(page2Again.items[0], user4);
         assert.deepEqual(page2Again.items[1], user5);
         assert.isDefined(page2Again.firstEvaluatedKey);
+        assert.equal(page2Again.firstEvaluatedKey, page2.firstEvaluatedKey);
         assert.isUndefined(page2Again.lastEvaluatedKey);
```

Comment:
Can we add assert.deepEqual(page2Again.lastEvaluatedKey, page2.lastEvaluatedKey) ?